lib/ansible/runner.rb

```diff
@@ -0,0 +1,29 @@
+module Ansible
+  class Runner
+    class << self
+      def run(env_vars, extra_vars, playbook_path)
+        run_via_cli(env_vars, extra_vars, playbook_path)
```

Comment:
@agrare I wonder if allowing any path is too much? Maybe we should pass ems and limit it to ems/ansible/ dir?

Reply:
@Ladas you mean from a security standpoint or usability?  I think we'll want to keep this generic, in the provider we might want to have a helper based on the engine root path which we won't know here.

Reply:
Actually what you have [here](https://github.com/ManageIQ/manageiq-providers-amazon/pull/453/files#diff-235184baadf210fc9a4d037d8ffa1c37R83) is exactly what I was thinking

Reply:
ok, sounds good